src/test/util_tests.cpp

```diff
@@ -6,18 +6,18 @@
 
 #include <clientversion.h>
 #include <hash.h> // For Hash()
-#include <key.h> // For CKey
+#include <key.h>  // For CKey
```

Comment:
Was this changed manually?

Reply:
No, it is part of the scripted-diff, which you can verify by running `./test/lint/commit-script-check`

Reply:
Thanks -- I need to upgrade my scripted-diff-fu